omod/src/test/java/org/openmrs/module/webservices/rest/doc/SwaggerSpecificationCreatorTest.java

```diff
@@ -9,31 +9,73 @@
  */
 package org.openmrs.module.webservices.rest.doc;
 
-import java.sql.Connection;
-import java.sql.DatabaseMetaData;
-import java.sql.ResultSet;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.converter.ModelConverterContextImpl;
+import io.swagger.converter.ModelConverters;
+import io.swagger.jackson.ModelResolver;
+import io.swagger.models.Info;
+import io.swagger.models.Model;
+import io.swagger.models.Operation;
+import io.swagger.models.Path;
+import io.swagger.models.Scheme;
+import io.swagger.models.Swagger;
+import io.swagger.models.auth.BasicAuthDefinition;
+import io.swagger.models.parameters.Parameter;
+import io.swagger.util.Json;
 import org.dbunit.database.DatabaseConnection;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 import org.openmrs.GlobalProperty;
+import org.openmrs.Patient;
 import org.openmrs.api.context.Context;
-import org.openmrs.module.webservices.docs.swagger.Operation;
-import org.openmrs.module.webservices.docs.swagger.Parameter;
-import org.openmrs.module.webservices.docs.swagger.Path;
-import org.openmrs.module.webservices.docs.swagger.SwaggerSpecification;
 import org.openmrs.module.webservices.docs.swagger.SwaggerSpecificationCreator;
 import org.openmrs.module.webservices.rest.web.RestConstants;
 import org.openmrs.module.webservices.rest.web.api.RestService;
 import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
 
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static junit.framework.TestCase.assertNotNull;
+
 public class SwaggerSpecificationCreatorTest extends BaseModuleWebContextSensitiveTest {
 	
+	@Test
+	public void mainTest() {
+		String baseUrl = "localhost:8080/openmrs/ws/rest";
```

Comment:
In either case, hard coding a port number is not good practice. Make BaseUrl generic if it doesn't make a difference